lib/octicons_react/__tests__/tree-shaking.test.js

```diff
@@ -0,0 +1,30 @@
+const commonjs = require('@rollup/plugin-commonjs')
+const {nodeResolve} = require('@rollup/plugin-node-resolve')
+const {rollup} = require('rollup')
+const virtual = require('@rollup/plugin-virtual')
+
+test('tree shaking', async () => {
+  const bundle = await rollup({
+    input: '__entrypoint__',
+    external: [],
+    plugins: [
+      nodeResolve(),
+      commonjs(),
+      virtual({
+        __entrypoint__: `import { AlertIcon } from '@primer/octicons-react'`
+      })
+    ],
+    onwarn: ({code, message}) => {
+      if (code !== 'EMPTY_BUNDLE') {
+        throw new Error(message)
+      }
+    }
+  })
+  const {output} = await bundle.generate({
+    format: 'esm'
+  })
+
+  for (const {code} of output) {
+    expect(code.trim()).toBe('')
```

Comment:
If I'm reading this correctly, this test asserts that unused imports don't get bundled. Do we also need to check that importing and use a single icon doesn't bring all the icons with it?

Reply:
@colebemis would be great to add, I'm just not sure what the assertion would look like 🤔 let me know if you have any ideas!